execution/executor-benchmark/src/main.rs

```diff
@@ -324,6 +326,20 @@ enum Command {
 
         #[clap(long, value_parser)]
         checkpoint_dir: PathBuf,
+
+        #[clap(
+            long,
+            num_args=1..,
+            value_delimiter = ' ',
+            help = "Custom enabling/disabling of the feature flags in the Move source. Disabling is stronger than enabling")]
+        enable_feature: Vec<FeatureFlag>,
+
+        #[clap(
+            long,
+            num_args=1..,
+            value_delimiter = ' ',
+            help = "Custom enabling/disabling of the feature flags in the Move source. Disabling is stronger than enabling")]
+        disable_feature: Vec<FeatureFlag>,
```

Comment:
ideally you want to check if there is any overlap between two lists, and report error, rather than having `disable_feature` blindly override the `enable_feature`

Reply:
Thanks, implemented!